src/javascript/javascript-parser.ts

```diff
@@ -27,6 +32,31 @@ declare class SyntaxError {
   column: number;
 }
 
+/**
+ * The `Comment` object created by `espree.parse` has some properties not
+ * included in `@types/estree` like `loc`, which is needed to calculate
+ * the source range.
+ */
+export interface Comment extends EstreeComment, BaseNode {
+  type: 'Block' | 'Line';
```

Comment:
Ok, how do you want to proceed here? Create a PR for estree and then come back to this one once it's merged?